packages/vapor/scss/variables.scss

```diff
@@ -525,13 +525,12 @@ $step-progress-bar-doing-color: var(--caribbean-green);
 $step-progress-bar-to-do-color: var(--white);
 
 // Badge
-$badge-font-size: 0.77em;
+$badge-font-size: 0.923em;
 $badge-mod-small-font-size: 9px;
 $badge-height: 2em;
 $badge-line-height: 1em;
-$badge-padding: 0.5em 1em;
+$badge-padding: 0.46em 1.23em 0.46em 0.92em;
```

Comment:
I'm a little curious about all these magic numbers lol

Reply:
I did a ratio of 1em/13px, so I converted the pixel dimensions back to em's here. The badge padding in pixels is `6px 16px 6px 12px`. I hope this is okay.
![image](https://user-images.githubusercontent.com/66333175/105528344-e02e5900-5cb2-11eb-9e97-0ca2a545096a.png)